build-tools/xaprepare/xaprepare/xaprepare.csproj

```diff
@@ -62,4 +62,10 @@
   </ItemGroup>
 
   <Import Project="xaprepare.targets" Condition=" $(MSBuildToolsPath.IndexOf('omnisharp')) &lt; 0 " />
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netcoreapp3.1' ">
+    <PackageDownload Include="Microsoft.NETCore.App.Runtime.android-arm64" Version="$(DotNetRuntimePacksVersion)" />
+    <PackageDownload Include="Microsoft.NETCore.App.Runtime.android-x64" Version="$(DotNetRuntimePacksVersion)" />
+    <PackageDownload Include="Microsoft.NETCore.App.Runtime.android-arm" Version="$(DotNetRuntimePacksVersion)" />
+    <PackageDownload Include="Microsoft.NETCore.App.Runtime.android-x86" Version="$(DotNetRuntimePacksVersion)" />
```

Comment:
If you get an error, you might need to add the `[]` to these lines.

Reply:
Yep, just pushed the change, thanks :)